src/commands/image/deployImageApi/deployImage.ts

```diff
@@ -45,4 +47,8 @@ export async function deployImage(context: IActionContext & Partial<IContainerRe
 
     await wizard.prompt();
     await wizard.execute();
+
+    if (!settingUtils.getSetting('suppressActivityNotifications', undefined, 'azureResourceGroups')) {
+        void showContainerAppNotification(containerApp, true /** isUpdate */);
```

Comment:
@aweininger Correct me if I'm wrong, but your intention for this [comment](https://github.com/microsoft/vscode-azurecontainerapps/pull/505#discussion_r1365859012) was that the `void showContainerAppNotification(containerApp, true /** isUpdate */);` gets moved into one of the wizard steps so that it would respect the suppress activity notifications setting, right?

Reply:
Whoops, meant to tag @alexweininger. No idea who that other dude is.